kubetest/client.py

```diff
@@ -971,7 +971,7 @@ def get_replicasets(
         replicasets = {}
         for obj in results.items:
             rs = objects.ReplicaSet(obj)
-            replicasets[replicasets.name] = rs
```

Comment:
without this change, the returned value would be {}

Reply:
`replicasets` is a dictionary which has no name property. I'm pretty sure this was just a typo